internal/cluster/transition.go

```diff
@@ -159,6 +159,9 @@ func createClusterCompletionStatusInfo(log logrus.FieldLogger, cluster *common.C
 		return statusInfoInstalled
 	}
 
+	msg := fmt.Sprintf("Cluster %s is installed but degraded due to failed OLM operators", cluster.Name)
```

Comment:
Please add the failed OLM operators same as we do in line 166: statusInfo += ". Failed OLM operators: " + strings.Join(statuses[models.OperatorTypeOlm][models.OperatorStatusFailed], ", ")

Reply:
+1